README.md

```diff
@@ -1 +1,38 @@
-# GoodBooks-ES6
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# AWESOME BOOKS
+
+> This website GoodBooks provides users a convenient way to keep track of their books or reading lists. Users can add and remove books from the list of books they curate on the website.
+
+<!-- ![App Screenshot]() -->
+
```

Comment:
- [ ] It would be better if you add how to clone the project and how to start the project. kindly add this. This will make you repo looks professional 🛠️ 😄